pkg/auth/policy.go

```diff
@@ -0,0 +1,100 @@
+package auth
```

Comment:
I was planning on adding authn to this package too, yes, because there will be some common components between the two (they are really both just a thin veneer on top of OPA). But if that proves unwieldly, I'll split them into separate packages.